static/css/restyle.less

```diff
@@ -0,0 +1,853 @@
+// Colors
+@primary-blue: rgba( 12, 153, 213, 1);
+@primary-link: #6AC3E2;
+
+#tabzilla:before {
+    background: none;
+}
+
+body {
+    font-family: "Open Sans",X-LocaleSpecific,sans-serif !important;
+    font-style: normal;
+    font-size: 0.875rem;
+    line-height: 1.5;
+    color: #333;
+    background: #FFF none repeat scroll 0% 0%;
+    min-width: 1018px;
+}
+
+h1 {
+    font-family: "Open Sans",X-LocaleSpecific,sans-serif !important;
+    font-style: normal;
+    font-size: 2.25rem;
+}
+
+/* Main Content Wrapper */
+#background-wrapper {
+    border-top: 2px solid #FFF;
+    background-color: #0C99D5;
+    width: 100%;
+    position: absolute;
+    height: 130px;
+    /* Main Content Wrapper Background - General
+     blue-pattern, blue-gradient, white overlay
+     (sharp-cut at 130px) */
+    background-image:
+    linear-gradient(
```

Comment:
This would read easier if intended, sorta like you did here: https://github.com/mozilla/olympia/pull/1078/files#diff-7162e806e2f47b405e38dccc33900c61R56